tests/test_climate_baseline_helpers.py

```diff
@@ -0,0 +1,169 @@
+import datetime
+from calendar import monthrange
+import struct
+
+from pytest import fixture, mark, raises
+
+import pycds
+from pycds.util import generic_sesh
+from pycds import Network, Station, History, Variable, DerivedValue
+from pycds.climate_baseline_helpers import \
+    pcic_climate_variable_network_name, \
+    get_or_create_pcic_climate_variables_network, \
+    create_pcic_climate_baseline_variables, \
+    load_pcic_climate_baseline_values, \
+    field_format, \
+    InvalidVariableName
+
+
+@fixture
```

Comment:
Is this fixture used in other testing modules? Should it be added to `conftest.py`?